.yarnrc.yml

```diff
@@ -1,11 +1,11 @@
+compressionLevel: mixed
+
+enableGlobalCache: false
```

Comment:
this was auto set by Yarn 4. I believe this was the default in Yarn 3 but not Yarn 4, so I didn't want to change it.